pytorch_lightning/trainer/training_tricks.py

```diff
@@ -19,35 +19,42 @@
 from torch import Tensor
 
 from pytorch_lightning.core.lightning import LightningModule
+from pytorch_lightning.utilities import rank_zero_deprecation
+from pytorch_lightning.utilities.nan import detect_nan_parameters, print_nan_gradients
 
 EPSILON = 1e-6
 EPSILON_FP16 = 1e-5
 log = logging.getLogger(__name__)
 
 
 class TrainerTrainingTricksMixin(ABC):
+    """
+    TODO: This class will be deprecated in v1.5.
```

Comment:
Is this what you meant?

```suggestion
    TODO: Remove this class in v1.5.
```

Reply:
rather make `#todo: ...` above the function, so IDE better index todos